client/src/views/Home/HowToPlay.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
```

Comment:
HowToPlay.jsx should be HowToPlayModal.jsx and placed in the components directory.